### PR TITLE
added decimal extension for round

### LINKED ISSUE #16: Add decimal extension for rounding
Add decimal extension for rounding

### PR DESCRIPTION
**Pull request by Michalau**
Closes #16

### CODE REVIEW

File: tests/Extensions/DecimalExtensionsShould.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+{
+    public class DecimalExtensionsShould
+    {
+        [Fact]
+        public void ReturnNumberRoundedDown()
+        {
+            decimal input = 10.124m;
+            decimal actual = input.Round(2);
+            decimal expected = 10.12m;
+            Assert.Equal(expected, actual);
```
Comment (Burgyn) [inline tests/Extensions/DecimalExtensionsShould.cs]: We don't use standard assertion system. Use [Fluent Assertion](https://fluentassertions.com/introduction) instead.

File: tests/Extensions/DecimalExtensionsShould.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
+﻿using Kros.Extensions;
+using Xunit;
+
+namespace Kros.Utils.UnitTests.Extensions
+{
+    public class DecimalExtensionsShould
+    {
+        [Fact]
```
Comment (Burgyn) [inline tests/Extensions/DecimalExtensionsShould.cs]: I suggest using `[Theory]` instead of `[Fact]` and adding more test data.

**Review by Burgyn — CHANGES_REQUESTED** [review]

**Comment by satano** [comment]
Do we really want such a specific extensions here? Thes library was intended as general purpose.

If yes, I would make extension methods for all rounding methods (`decimal` and `double`, and without the need of number of decimal places). Maybe even with the posibility to specify `MidpointRounding`. So we will have all `Math.Round` methods, just the default for `MidpointRounding` will be the other one.

File: tests/Extensions/DecimalExtensionsShould.cs (hunk 1 above)
Comment (Michalau) [reply tests/Extensions/DecimalExtensionsShould.cs]: Resolved

File: tests/Extensions/DecimalExtensionsShould.cs (hunk 2 above)
Comment (Michalau) [reply tests/Extensions/DecimalExtensionsShould.cs]: Resolved

**Review by satano — APPROVED** [review]

**Pull request merged**